proposals/lip-0053.md

```diff
@@ -102,9 +101,9 @@ The role of relayer is totally symmetric for relaying information from a sidecha
 
 The Lisk protocol allows relayers to post cross-chain update transactions which do not contain all CCMs, as opposed to the mechanism explained in the previous section. This can be useful for example when the list of CCMs is too large to be included in a single block.
 
-The cross-chain update transaction can contain an `inboxUpdate` which does not contain all CCMs required to recompute the outbox root corresponding to the given state root. In that case, the relayer has to make sure that an appropriate `messageWitness` is provided. This witness is constructed as a right witness as defined in [LIP 0031](https://github.com/LiskHQ/lips/blob/main/proposals/lip-0031.md).
+The cross-chain update transaction can contain an `inboxUpdate` which does not contain all CCMs required to recompute the outbox root corresponding to the given state root. In that case, the relayer has to make sure that an appropriate `messageWitnessHashes` is provided. This witness is constructed as a right witness as defined in [LIP 0031](https://github.com/LiskHQ/lips/blob/main/proposals/lip-0031.md).
 
-If a cross-chain update was submitted without all the CCMs (as explained in the previous paragraph), then it is possible to provide those missing CCMs without the need to provide another `certificate` or `outboxRootWitness` (technically an empty `certificate` and `outboxRootWitness` are provided). The cross-chain update transaction contains only the missing CCMs and the potential `messageWitness`.
+If a cross-chain update was submitted without all the CCMs (as explained in the previous paragraph), then it is possible to provide those missing CCMs without the need to provide another `certificate` or `outboxRootWitness` (technically an empty `certificate` and `outboxRootWitness` are provided). The cross-chain update transaction contains only the missing CCMs and the potential `messageWitnessHashes`.
 
 Those options for partial cross-chain update transactions are not expected to be used regularly. It is however a good option in corner case events where the sidechain outbox suddenly becomes very large.
 
```

Comment:
Isn't this section "Malicious Cross-chain Update Transactions" rather a new section than a subsection of "Creating and Posting Cross-chain Update Transactions". Further, I find the word malicious misleading in this context as the CCU itself is a valid transaction and also the relayer may not be a malicious entity. Invalid CCMs show rather a violation of the interoperability protocol that should only happen if someone modifies the interoperability module.

Reply:
same as below

Reply:
I would still change "Malicious Cross-chain Update Transactions" to "Invalid Cross-Chain Messages Imply Termination" or something similar. Or the content of this section could be moved to the section "Cross-chain Update Command Processing Stages".

Reply:
I renamed it to "Invalid Cross-Chain Messages"